ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -92,15 +104,53 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
 		containerDefinitions = append(containerDefinitions, containerDef)
 	}
 
+	var ecsFields *ECSFields
+	if ecsFieldsFileName != "" {
+		ecsFields, _ = readECSFields(ecsFieldsFileName)
```

Comment:
Why is the error being ignored here?

Reply:
Good catch. Residual ignoring from something else. Fixed.